mainapp/templates/account.html

```diff
@@ -60,6 +78,97 @@ <h4 class=" mb-0">
         </div>
     </div>
 
+    <!-- Landlord Apartments -->
+    {% if user.landlord_apartment_set %}
+        <hr>
+        <h1>Landlord Apartments</h1>
+        <!--Section: Cards-->
+        <section class="text-center">
+        {% for apt in user.landlord_apartment_set.all %}
+            {% cycle 'odd' 'even' as odd_even silent %}
+            {% if odd_even == "odd" %}
+            <!--Grid row-->
+            <div class="row mb-4 wow fadeIn">
+            {% endif %}
+            <!--Grid column-->
+            <div class="col-md-6 mb-4">
+            <!--Card-->
+            <div class="card">
```

Comment:
Would it be possible to make the cards the same size? Otherwise it could be an issue if one apartment photo is drastically larger than the other.

![image](https://user-images.githubusercontent.com/32490273/69197390-31671980-0aff-11ea-8e57-e52aaa970d51.png)

Reply:
Thank you for the suggestion! That was important. Fixed!